src/Modules/Build.lua

```diff
@@ -156,11 +156,53 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
 		end
 	end
 	self.controls.pointDisplay.width = function(control)
-		local used, ascUsed = self.spec:CountAllocNodes()
-		local usedMax = 99 + 22 + (self.calcsTab.mainOutput.ExtraPoints or 0)
-		local ascMax = 8
-		control.str = string.format("%s%3d / %3d   %s%d / %d", used > usedMax and "^1" or "^7", used, usedMax, ascUsed > ascMax and "^1" or "^7", ascUsed, ascMax)
-		control.req = "Required level: "..m_max(1, (100 + used - usedMax))
+		local PointsUsed, AscUsed = self.spec:CountAllocNodes()
+		local bandit = self.calcsTab.mainOutput.ExtraPoints or 0 
+		local usedMax, ascMax, levelreq, currentAct, banditStr, labSuggest = 99 + 22 + bandit, 8, 1, 1, "", ""
+		local acts = { 
+			[1] = { level = 12, questPoints = 2}, 
+			[2] = { level = 22, questPoints = 3}, 
+			[3] = { level = 32, questPoints = 5},
+			[4] = { level = 40, questPoints = 6},
+			[5] = { level = 44, questPoints = 8},
+			[6] = { level = 50, questPoints = 11},
+			[7] = { level = 54, questPoints = 14},
+			[8] = { level = 60, questPoints = 17},
+			[9] = { level = 64, questPoints = 19},
+			[10] = { level = 67, questPoints = 22}
+			}
+				
+		-- loop for how much quest skillpoints are used with the progress
+		repeat
+		  if PointsUsed > acts[currentAct].level+acts[currentAct].questPoints then 
+			currentAct=currentAct+1 
+		  else 
+			break 
+		  end
+		until currentAct>=10
```

Comment:
This would be more succinct and accomplish the same goal:
```suggestion
		while currentAct < 10 and PointsUsed > acts[currentAct].level + acts[currentAct].questPoints do
			currentAct = currentAct + 1
		end
```